src/mono/wasm/runtime/debug.ts

```diff
@@ -45,7 +45,7 @@ export function mono_wasm_runtime_ready(): void {
         // eslint-disable-next-line no-debugger
         debugger;
     else
-        console.debug("mono_wasm_runtime_ready", "fe00e07a-5519-4dfe-b35a-f867dbaf2e28");
+        console.assert(true, "mono_wasm_runtime_ready", "fe00e07a-5519-4dfe-b35a-f867dbaf2e28");
```

Comment:
is assert available everywhere?

Reply:
It does not work on firefox, trying to find another solution.

Reply:
The browser API is everywhere. How it's reported to debugger protocol is different for FF.
https://caniuse.com/mdn-api_console_assert